internal/core/graphics/image.rs

```diff
@@ -454,6 +454,7 @@ impl PartialEq for ImageInner {
             (Self::StaticTextures(l0), Self::StaticTextures(r0)) => l0 == r0,
             #[cfg(target_arch = "wasm32")]
             (Self::HTMLImage(l0), Self::HTMLImage(r0)) => vtable::VRc::ptr_eq(l0, r0),
+            (Self::BackendStorage(l0), Self::BackendStorage(r0)) => vtable::VRc::ptr_eq(l0, r0),
             #[cfg(not(target_arch = "wasm32"))]
             (Self::BorrowedOpenGLTexture(l0), Self::BorrowedOpenGLTexture(r0)) => l0 == r0,
             _ => core::mem::discriminant(self) == core::mem::discriminant(other),
```

Comment:
maybe that's not so smart for future-proofness when we add more variant.

I suggest this:
```suggestion
            (Self::None, Self::None) => true,
            _ => false,
```

that way new variant will always be false which is better. (better have false negative than false positive.)